src/site/content/en/blog/samesite-cookies-explained/index.md

```diff
@@ -228,11 +243,14 @@ cookie. This makes `Lax` a good choice for cookies affecting the display of the
 site with `Strict` being useful for cookies related to actions your user is
 taking.
 
-{% Aside 'caution' %} Neither `Strict` nor `Lax` are a silver bullet for your
-site security. Cookies are sent as part of the user's request and you should
-treat them the same as any other user input. That means sanitizing and
-validating the input. Never use a cookie to store data you consider a
-server-side secret. {% endAside %}
+{% Aside 'caution' %}
+
+Neither `Strict` nor `Lax` are a silver bullet for your site security. Cookies
```

Comment:
'are a silver bullet' --> 'are silver bullets'

Actually, I'd probably replace this metaphor with 'cure'. Idioms don't translate well to other languages.

Reply:
Good point! I was thinking of:
> Neither `Strict` nor `Lax` are a complete solution for your site's security.

Reply:
Sounds good.